Portfolio Project - Saving Data on Local storage

davidolanrewaju opened the pull request:
## Saving Data on Local storage
>This PR contains the addition of logic to the pages contact form, to save inputted data on local storage for easy filling of forms

## 🛠Built With:
- HTML
- CSS
- JS
- GitHub Flows
- Linters

## ♻ Feature Update
- [x] Addition of logic to save form data on local storage for auto filling

davidolanrewaju committed:
Update HTML file

davidolanrewaju committed:
Save to local storage logic

davidolanrewaju committed:
Add external js link for storage to html file

davidolanrewaju committed:
HTML Update

m05e5 commented on storage.js:
```diff
@@ -0,0 +1,27 @@
+// Get references to the form inputs
+const nameInput = document.getElementById('name');
+const emailInput = document.getElementById('email');
+const messageInput = document.getElementById('message');
```
- [ ] Hi as part of the requirements, you're supposed to preserve data in the form and redisplay it after the page reloads but currently, it's not working. I realized you are trying to get the element with id `name` in the form but in your HTML file you don't have any element of id `name` Please kindly fix it
<img width="1300" alt="image" src="https://user-images.githubusercontent.com/37901469/224303673-b43da8e4-feb8-4983-817f-48d7091c2166.png">

m05e5 reviewed (changes requested):
Hi Team , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
![mario_ladders](https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif)


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

joyapisi committed:
add correct id to name input in contact form

add correct id to name input in contact form and asterisks to show that content is required

levy002 reviewed (approved):
## Hello @dave-prog 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://media.giphy.com/media/jJQC2puVZpTMO4vUs0/giphy.gif)

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
